yarn.lock

```diff
@@ -17245,7 +17245,7 @@ moo-color@^1.0.2:
 
 motion-ui@cultureamp/motion-ui:
   version "2.0.3"
```

Comment:
Interestingly the "version" hasn't changed. I think this is because we didn't tag any commits in our fork, so it's just the latest tag that happens to be there (which doesn't actually match the resolved github url). I downloaded `https://codeload.github.com/cultureamp/motion-ui/tar.gz/d73f199d0ab2626a334ba6100604b3d14e6dabf1` and checked it's the latest version.

Reply:
Also the repo still calls itself "2.0.3": https://github.com/cultureamp/motion-ui/blob/master/package.json#L3